src/Cassandra/HostConnectionPool.cs

```diff
@@ -343,6 +345,8 @@ internal Task<Connection[]> MaybeCreateCorePool()
             {
                 if (t.Status != TaskStatus.RanToCompletion)
                 {
+                    if (t.Exception != null)
+                        t.Exception.Handle(e => true);
```

Comment:
small nit: can we add `{` and `}` for the statement, even if its a one-liner, like the rest of the file.